src/searchModal/advancedSearch.js

```diff
@@ -91,11 +91,14 @@ export default function advancedSearchFactory(config) {
         /**
          * Builds substring of search query with the advanced criteria conditions
          */
-        getAdvancedCriteriaQuery: function () {
+        getAdvancedCriteriaQuery: function (hasSearchInput) {
             const advancedSearchCriteria = _.filter(criteriaState, criterion => criterion.rendered === true);
             let query = '';
 
             advancedSearchCriteria.forEach(renderedCriterion => {
+                if (hasSearchInput || query.trim().length !== 0) {
+                    query += ? " AND "
```

Comment:
**nitpick:** is it a typo?

**chore:** the ending `;` is missing.

**suggestion:**
```suggestion
                    query += " AND ";
```

Reply:
omg, I forget conventional commits -.-. Already fixed. Wait because I will push some new unit tests :) (with the test flag in conventional commits)